.ci/download-deps.sh

```diff
@@ -24,15 +29,33 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 # THE POSSIBILITY OF SUCH DAMAGE.
+#;**********************************************************************;
+
+function get_deps() {
 
-# This script is a wrapper around the the 'make install' command for the
-# TSS source code. This is a workaround for a travis-ci quirk that causes
-# any command run under 'sudo' to be unable to run clang. This script fixes
-# up the borked PATH, moves to the directory where we built the TSS and then
-# runs 'make install'
-# see: https://github.com/travis-ci/travis-ci/issues/3088
+	echo "pwd starting: `pwd`"
+	pushd /workspace
+	echo "pwd clone tss: `pwd`"
+	git clone https://github.com/tpm2-software/tpm2-tss.git
+	pushd tpm2-tss
+	echo "pwd build tss: `pwd`"
+	./bootstrap
+	./configure
+	make -j4
+	make install
+	popd
+	echo "pwd done tss: `pwd`"
 
-PATH=${PATH}:/usr/local/clang/bin
-(pushd ${TRAVIS_BUILD_DIR}/tpm2-tss && \
- make install && \
- popd)
+	echo "pwd clone abrmd: `pwd`"
+	git clone https://github.com/tpm2-software/tpm2-abrmd.git
+	pushd tpm2-abrmd
+	echo "pwd build abrmd: `pwd`"
+	./bootstrap
+	./configure --with-dbuspolicydir=/etc/dbus-1/system.d
```

Comment:
We put a conf file in the image in the right spot that doesn't have the "tss" user, so delete this.